src/platform/user/authentication/utilities.js

```diff
@@ -285,18 +285,41 @@ export function redirect(redirectUrl, clickedEvent, type = '') {
   window.location = redirectUrl;
 }
 
+export async function mockLogin({ clickedEvent = AUTH_EVENTS.MOCK_LOGIN }) {
+  const { application, codeChallenge, codeChallengeMethod } = getQueryParams();
+  const gaClientId = '';
+  const config =
+    externalApplicationsConfig[application] ||
+    externalApplicationsConfig.default;
```

Comment:
? Can you explain why this is needed?

Reply:
Was added to mirror code throughout this section, but upon testing it's deemed unnecessary so removed.